src/TouristInfoCards.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const TouristInfoCards = props => {
+  return (
+    <div className="xs-col-12 sm-col-4">
+      <div class="card" style={{ width: '18rem' }}>
```

Comment:
You've got the one above correct, but remember that you have to use `className` instead of `class` when using React!